N/NCCL/build_tarballs.jl

```diff
@@ -7,43 +7,48 @@ include(joinpath(YGGDRASIL_DIR, "fancy_toys.jl"))
 include(joinpath(YGGDRASIL_DIR, "platforms", "cuda.jl"))
 
 name = "NCCL"
-version = v"2.19.4"
+version = v"2.26.5"
+
+MIN_CUDA_VERSION = v"11.4" # previous min was 11.4
 
 # Collection of sources required to complete build
 sources = [
-    GitSource("https://github.com/NVIDIA/nccl.git", "88d44d777f6970bdbf6610badcbd7e25a05380f0"),
-    DirectorySource("./bundled"),
+    GitSource("https://github.com/NVIDIA/nccl.git", "3000e3c797b4b236221188c07aa09c1f3a0170d4"),
 ]
 
-# Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir
 
 export TMPDIR=${WORKSPACE}/tmpdir # we need a lot of tmp space
-export CUDA_HOME=${WORKSPACE}/destdir/cuda
-export CUDA_LIB=${CUDA_HOME}/lib
+mkdir -p ${TMPDIR}
+
 export CXXFLAGS='-D__STDC_FORMAT_MACROS'
 export CUDARTLIB=cudart # link against dynamic library
 
-mkdir -p ${TMPDIR}
+export CUDA_HOME=${prefix}/cuda;
+export PATH=$PATH:$CUDA_HOME/bin
+export CUDACXX=$CUDA_HOME/bin/nvcc
+
+# nvcc/nccl thinks the libraries are located inside lib64, but the SDK actually has them in lib
+ln -s ${CUDA_HOME}/lib ${CUDA_HOME}/lib64
 
 cd nccl
+make -j ${nproc} src.build CUDA_HOME=${CUDA_HOME} PREFIX=${prefix}
```

Comment:
Looking at my build history in https://github.com/JuliaPackaging/Yggdrasil/pull/7794, I tried this originally, but for some reason I decided the `pkg.txz.build` path was easier.